CMakeLists.txt

```diff
@@ -42,6 +42,37 @@ add_library(
   TritonCommon::triton-common-json ALIAS triton-common-json
 )
 
+add_library(
```

Comment:
You put this in the middle of the triton-common-json setting. Move it all to just before install